src/test/java/EulerOneTest.java

```diff
@@ -0,0 +1,72 @@
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class EulerOneTest {
+    @Test
+    void whenSetMaxRangeTo10_Give23(){
```

Comment:
Zamiast `Give` raczej stosujemy `Then` ale ciężko coś mówić, bo tak jak mówiłem na zajęciach - to jest wszystko kwestia konwencji przyjętej w projekcie.